public/index.html

```diff
@@ -63,6 +63,9 @@ <h1 class="title display-2 fw-bolder">GNU/Weeb</h1>
               <a class="gw-button" target="_blank" href="//git.io/GNUWeeb">
```

Comment:
@Sprtcrnbry can you change the link (telegram and Github) to `https://` too?
I previously used `//` instead of `https: //`?

Reply:
I am addressing your review. Please wait for it.

Reply:
im not sure whats the difference, but im sure by using `https://` it forces https instead (more secure).

Reply:
@Sprtcrnbry using // will adapt to the current page scheme, if it uses HTTPS then it be HTTPS, the same happens with HTTP.

And yes, using https:// forces it to be HTTPS URL regardless accessed scheme.

Reply:
I am resolving this review.